documentation/modules/auxiliary/admin/http/ulterius_file_download.md

```diff
@@ -0,0 +1,79 @@
+## Overview
+
+This module exploits a directory traversal vulnerability in [Ulterius Server < v1.9.5.0](https://github.com/Ulterius/server/releases). The directory traversal flaw occurs in Ulterius Server's HttpServer.Process function call. While processing file requests, the HttpServer.Process function does not validate that the requested file is within the web server's root directory or a subdirectory.
```

Comment:
i'd throw ticks around HttpServer.Process so it looks more like a function and less like a strange period (end of sentence).

Reply:
Changed. I also added ticks around fileIndex.db for the same reasoning.